recipes/pymeos/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pymeos" %}
+{% set version = "1.1.3b3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 363b5fe10314f481f71361ed9e6f21443e4bf94c2546e4bdc1d31ffa70c85fce
+
+build:
+  skip: true  # [not linux]
+  skip: true  # [py < 37]
```

Comment:
This one can probably be noarch.

Reply:
I think so too, but I set it like this since it's the same as the `pymeos_cffi` package which this depends on. I can change it if you think it's better

Reply:
It is fine to keep this noarch and let Windows and mac-OS fail with a missing dependency , or at least OS-noarch. That way we avoid the Python build matrix and save some CI resources.

Reply:
I've made it noarch and moved the python constraints into the host and run sections following the docs at https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds